src/daemon/taproxy.rs

```diff
@@ -736,6 +767,23 @@ impl TrustAnchorProxy {
         }
     }
 
+    fn process_update_signer(
+        &self,
+        signer: TrustAnchorSignerInfo,
+    ) -> KrillResult<Vec<TrustAnchorProxyEvent>> {
+        if let Some(s) = &self.signer {
+            if s.ta_cert_details.cert().key_identifier() == 
+                signer.ta_cert_details.cert().key_identifier() {
```

Comment:
Nitpick: The opening brace should be on its own line if the condition has a line break.